_docs-sources/pipelines/initial-setup/index.md

```diff
@@ -0,0 +1,20 @@
+# Initial setup
+
+TODO: Translate this into prose, or headers
+
+- You'll need two repos:
+  - infra-live
+  - infra-pipelines
+- In the infra-pipelines repo:
+  - Create the following GitHub Actions
+  - Setup AWS OIDC
+- In the infra-live repo:
+  - Create the following GitHub Actions
+- Now make your first change in the infra-live repo
+  - You should see Pipelines run pipelines actions
+- To configure these actions, do xxx...
+
+Alternatively:
+
+- For now, Gruntwork configures Pipelines for you.
+- We'll publish official instructions on how to set it up shortly.
```

Comment:
This section is key because now new customers can "just set it up" once they get access!

Reply:
Are you going to write this section?

Reply:
I actually wouldn't know how! But it's probably the single most actionable doc in the whole section, so we should have some plan for getting it written.